libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -229,9 +235,7 @@ export class OperationExecutionManager {
 
     await this._beforeExecuteOperations?.(this._executionRecords);
 
-    // This function is a callback because it may write to the collatedWriter before
-    // operation.executeAsync returns (and cleans up the writer)
-    const onOperationCompleteAsync: (record: OperationExecutionRecord) => Promise<void> = async (
+    const beforeOperationResult: (record: OperationExecutionRecord) => Promise<void> = async (
```

Comment:
```suggestion
    const beforeOperationResultAsync: (record: OperationExecutionRecord) => Promise<void> = async (
```
Per our standard naming convention in the repository.